rho/scancommand.py

```diff
@@ -86,6 +86,38 @@ def redacted_auth_string(auth):
         name, username, ssh_key_file)
 
 
+def redact_dict(redact_key_list, a_dict):
+    """redact_values in a dictionary
+    :param redact_key_list: keys in dictionary that value should be redacted
+    :param a_dict: A dictionary to redact
+    :return dictionary
+    """
+    for key in redact_key_list:
+        if a_dict is not None and key in a_dict:
+            a_dict[key] = '*******'
```

Comment:
Is this replacing the other places where the passwords are masked? Maybe we can create a constant  so if we need to change the masked representation in the future it will be easier.

I mean this can be used (later) on the other places that the literal password masking is done.